AnimatedTextInput/Classes/AnimatedTextInput.swift

```diff
@@ -170,7 +170,7 @@ open class AnimatedTextInput: UIControl {
         }
     }
 
-    fileprivate let lineView = AnimatedLine()
+    public let lineView = AnimatedLine()
```

Comment:
can't be this solve just by making an open var?? e.g.

```swift
    open var lineViewHidden: Bool = false {
        didSet {
            if lineViewHidden {
                lineView.fillLine(with: UIColor.clear)
            } else {
                lineView.fillLine(with: style.lineActiveColor)
            }
        }
    }

```

Reply:
Sure, that will work as well. 
My concern about making `lineView` public was that we would be exposing implementation details that should be private.